js/script.js

```diff
@@ -0,0 +1,179 @@
+/*
+* Ryne Schillinger &
+* Congcong Li
+*/
+
+
+$(document).ready(function() {
+
+	// STARTING THE QUIZ
+
+	$("#title-container").show();
+	$("#questions-container").hide();
+	$("#score-container").hide();
+	$("#1").prop("checked", true)
+
+	$("#start").click(function(event) {
+		$("#title-container").hide();
+		$("#questions-container").show();
+	});
+
+
+	// GENERATING THE QUESTIONS
+
+	class question {
+	  constructor(question, answer1, answer2,answer3,answer4,correctAnswer) {
+	    this.question = question;
+		this.answers = [answer1, answer2, answer3, answer4];
+	    this.correctAnswer=correctAnswer;
+	  }
+	}
+
+	var questions = [];
+
+	questions.push(new question(
+		"''Most rappers' taste level ain't at my waist level / Turn up the bass 'til it's ______ level''",
+		"Somewhere-in-space",
+		"Up-in-yo-face",
+		"Just-a-disgrace",
+		"Give-me-some-space",
+		1));
+
+	questions.push(new question(
+		"''Maybe 15 minutes, took some pictures with your sister / ______, then I'm finished, then it's back to business''",
+		"Get forgiveness", 
+		"Get a witness", 
+		"Merry Christmas", 
+		"Get some distance", 
+		2));
+
+	questions.push(new question(
+		"''You like piña coladas, getting caught in the rain / Or rocking flannels all summer like ______''",
+		"You're from the Ukraine",
+		"Jack Lalanne",
+		"Kurt Cobain",
+		"Garth and Wayne",
+		2));
+
+	questions.push(new question(
+		"''Her man in the store tryna try his best / But he just can't seem to ______''",
+		"Find the right vest",
+		"Catch Kanye West",
+		"Get any press",
+		"Get Kanye fresh",
+		3));
+
+	questions.push(new question(
+		"''Break records at Louis, ate breakfast at Gucci / My girl a superstar all from ______''",
+		"A home movie",
+		"Selling jewelry",
+		"Juice getting off scot-free",
+		"Starring in a nudie",
+		0));
+
+
+	// DISPLAYING THE CURRENT QUESTION
+
+	function showQuestion(current) {
+		$("#question-title").text(current.question);
+		$("#answer1").text(current.answers[0]);
+		$("#answer2").text(current.answers[1]);
+		$("#answer3").text(current.answers[2]);
+		$("#answer4").text(current.answers[3]);
+	}
+
+
+	// TRACKING QUIZ PROGRESS
+
+	var choice = 0;
+	var numCorrect=0;
+	var numQuestion=0;
+
+	$("#numQuestion").text(numQuestion+1);
+	$("#totalQuestions").text(questions.length);
+
+
+	// CHOOSING AN ANSWER
+
```

Comment:
//Change line 98-113
var choiceSet=function{
	for (var i = 1; i <= questions.length; i++) {
		
		$("#i").click(function () {
		  choice = i-1;
		});
	}
}